### PR TITLE
Debounced nav

### PR DESCRIPTION
**Pull request by abaransy**
made nav bar bounce down when scrolling

### CODE REVIEW

File: frontend/src/components/nav/navbar.jsx [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+import React, { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { logout } from '../../actions/session_actions';
-import { NavLink } from 'react-router-dom';
+import { NavLink, Link } from 'react-router-dom';
 import { clearSessionErrors } from '../../actions/session_actions';
+import { Route, Redirect } from 'react-router-dom';
 import logo from '../../images/logo.png'
 import styled from 'styled-components';
+import regeneratorRuntime from "regenerator-runtime";
```
Comment (philgresh) [inline frontend/src/components/nav/navbar.jsx:9]: Is this library used?

**Comment by abaransy** [comment]
Async functions wouldn’t work in my las project without using this library, I think it’s necessary, but I’ll try removing it and seeing if anything explodes.

File: frontend/src/components/session/session_container.jsx [hunk 2]
```diff
@@ -1,43 +1,64 @@
...
+      await signupDispatch(user);
     }
+    if (user) props.history.replace("/");
   };
 
-  const processDemoForm = (user) => {
-    loginDispatch(user);
+  const processDemoForm = async (user) => {
+    await loginDispatch(user);
+    if (demoLogin) props.history.replace("/");
```
Comment (philgresh) [inline frontend/src/components/session/session_container.jsx:41]: prefer `history.push` (closer to being immutable)

File: frontend/src/components/nav/navbar.jsx [hunk 3]
```diff
@@ -87,27 +92,39 @@ const Logo = styled.img`
   height: 2rem; 
 `
 
-export default function Navbar({ stick, ele }) {
+export default function Navbar({sticky, ele}) {
+  const [out, setOut] = useState(false); 
   const dispatch = useDispatch();
   const loggedIn = useSelector((state) => state.session.isAuthenticated);
```
Comment (philgresh) [inline frontend/src/components/nav/navbar.jsx:98]: I think you had this selector elsewhere, might be worth pulling out into a `selectors` file

**Review by philgresh — APPROVED** [review]

**Comment by abaransy** [comment]
I’ll look into that

**Pull request merged**